allvers/1999-02-02-GPL-2.0-from-gnu.org.txt.notes

```diff
@@ -0,0 +1,4 @@
+Links:
+    - http://web.archive.org/web/19990202012404/http://www.gnu.org/copyleft/gpl.txt
```

Comment:
This is exactly the same as 1997-07-15 from GNU grep. In fact, this same URL is mentioned in the notes for that one as well ;-)